README.md

```diff
@@ -517,15 +521,15 @@ module.exports = function (migration) {
 };
 ```
 
-#### `changeEditorInterface (fieldId, widgetId[, settings])` : void
+#### `changeFieldControl (fieldId, widgetId[, settings, widgetNamespace])` : void
 
-Changes the editor interface of given field's ID.
+Changes control interface of given field's ID.
 
 **`fieldId : string`** – The ID of the field.
 
 **`widgetId : string`** – The new widget ID for the field. See the [editor interface documentation](https://www.contentful.com/developers/docs/concepts/editor-interfaces/) for a list of available widgets.
 
-**`settings : Object`** – Widget settings, with the following options:
+**`settings : Object`** – Widget settings and extension instance parameters. For builtin widgets, the the following options are available:
```

Comment:
I'd define that for instance parameters `settings` are a free-form object with values being numbers, booleans or strings.

Reply:
According to alex they could be objects as well, therefore they are defined as any. is this incorrect?

Reply:
let's double check API schema. I might be wrong

Reply:
https://github.com/contentful/content_api/blob/3e4bf6b4a7f648287fa8da3236b55c447190efb9/lib/handler/constraints/management/schemas/shared.js#L35

This states number, boolean or string